src/Confix.Tool/src/Confix.Tool/Middlewares/Encryption/Providers/AzureKeyVault/AzureKeyVaultEncryptionProviderConfiguration.cs

```diff
@@ -0,0 +1,15 @@
+using System.Text.Json;
+using System.Text.Json.Nodes;
+using ConfiX.Extensions;
+
+namespace Confix.Tool.Middlewares.Encryption.Providers.AzureKeyvault;
+
+public sealed record AzureKeyVaultEncryptionProviderConfiguration(
+    string? Uri,
+    string? KeyName,
+    string? KeyVersion
+)
```

Comment:
this is not how .net formats things btw. Even if there is no prettier, the formatting should be consistent.